File: python/1013.py
valores = input()

(a, b, c) = valores.split()

a_1 = int(a)
b_2 = int(b)
c_2 = int(c)

maior = (a_1 + b_2 + abs(a_1 - b_2)) // 2
maior_dos3 = (maior + c_2 + abs(maior - c_2)) // 2

print(maior_dos3, "eh o maior")

File: c/1013.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
  int a, b, c;
  scanf("%d %d %d", &a, &b, &c);

  int maior_ab = (a + b + abs(a - b)) / 2;

  int maior;

  if (maior_ab >= c)
  {
    maior = maior_ab;
  }
  else
  {
    maior = c;
  }

  printf("%d eh o maior\n", maior);

  return 0;
}
